jekyll/_cci2/runner-faqs.adoc

```diff
@@ -120,32 +128,198 @@ circleci runner token create <resource-class-name> <nickname>
 
-[#how-long-do-inactive-self-hosted-runners-persist-in-the-self-hosted-runner-inventory-page]
-== How long do inactive self-hosted runners persist in the self-hosted runner inventory page?
+[#container-runner-specific-faqs]
+== Container runner specific FAQs
 
-If a self-hosted runner has not contacted CircleCI in 12 hours, it will not show up in the inventory page on the https://app.circleci.com/[CircleCI web app].
+This section answers frequently asked questions for the CircleCI’s container runner product.
 
-[#how-can-i-tell-whether-a-host-with-a-self-hosted-runner-installed-is-executing-a-job]
-== How can I tell whether a host with a self-hosted runner installed is executing a job?
+[#only-one-resource-class-allowed-per-container-agent-deployment]
+=== Is there only one resource class allowed per container runner deployment?
 
-The recommended approach at this time is to query the host with the following command: 
+No, you can use as many resource classes as you desire with your container runner deployment. At least one resource class is required in order to run a job successfully with container runner.
+
+[#does-container-agent-use-a pull-model]
+=== Does container runner use a pull or push based model?
+
+Container runner uses a pull-based model.
+
+[#does-container-agent-scale-my-kubernetes-cluster]
+=== Does container runner scale my Kubernetes cluster for me?
+
+Container runner itself is its own deployment of a single replica set that does not currently require scaling. container runner will not scale the Kubernetes cluster itself. It schedules work if there are available resources in the cluster.
+
+As the technology is still in its early phases, the upper bound of how many concurrent tasks container runner can schedule without unforeseen issues is still being tested.
+
+You can use the <<runner-scaling#,queue depth API>> as a signal for cluster scaling.
+
+[#limit-for-the-number-of-concurrent-tasks]
+=== Is there a limit for the number of concurrent tasks that container runner can handle? 
+
+Container runner will claim and schedule work up to your runner concurrency limit. Additionally, by default, container runner is configured with a limit of 20 tasks it will allow to be concurrently scheduled and running.This can be configured via Helm to be a different value if your runner concurrency allows for a value greater than 20. See the `agent.maxConcurrentTasks` parameter in the <<#parameters,Parameters>> section above.
+  
+An organization’s runner concurrency limit is shared with any existing `machine` self-hosted runners. If you do not know what your organization's runner concurrency limit is, ask your point of contact at CircleCI, or submit a link:https://support.circleci.com/hc/en-us[support ticket].
+
+[#build-docker-images-with-container-agent]
+=== Can I build Docker images with container runner either via Remote Docker or Docker in Docker (DIND)?
```

Comment:
I think this might be contextual. There is both Remote Docker the product as well as the concept of a remote Docker which wouldn't be capitalized. When I google the phrase that seems to be the case as well.